manifest.yml

```diff
@@ -2,6 +2,8 @@ name: netlify-plugin-a11y
 inputs:
   - name: checkPaths
     required: true
+  - name: ignoreDirectories
+    required: false
```

Comment:
Could you please add `default: []` in `manifest.yml` and remove the default value `[]` from `index.js` and `pluginCore.js`?